### DISCUSSION TITLE
DDS example  code.

### DISCUSSION THREAD

**Discussion by paxifaer** [eProsima/Fast-DDS#5185] [Q&A]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

I found that the latest version of the sample code has changed a lot. It seems that the dds layer code of version 2.x.x has been deleted a lot. You have canceled many codes of QOS instances. Why? Will it be added later?
![screenshot-20240902-153042](https://github.com/user-attachments/assets/211417df-c02a-4ad1-ba4e-8676994aad6e)
![screenshot-20240902-152936](https://github.com/user-attachments/assets/d1be95b8-4ad9-48b7-a948-463ab4b4c5ec)

### Current behavior

lack of example code.

### Steps to reproduce

.

### Fast DDS version/commit

pre:2.x.x
now:3.0.0

### Platform/Architecture

Other. Please specify in Additional context section.

### Transport layer

Default configuration, UDPv4 & SHM

**Comment by rsanchez15** [comment]
Hi @paxifaer 

The examples have been refactored to make them easier to understand and test. Moreover, some old examples, such as all QoS examples, have been grouped together. You see fewer QoS examples in the root folder because all individual QoS examples are now inside the “_configuration_” example. Another improvement is the case of the transport configuration examples (SHM, TCP, Zero-Copy, etc.) which were previously separate and are now grouped under the “_delivery-mechanisms_” example.

**Comment by JesusPoderoso** [comment]
Hi @paxifaer, 
as part of the v3.0.0 release, all examples have been refactored to have a homogeneous structure, CLI usage, CLI output, documentation (README), and general behavior.
This [example section of the 2.x to 3.x migration guide](https://github.com/eProsima/Fast-DDS/blob/master/UPGRADING.md#examples) may clarify your doubts. As a quick answer, some redundant examples have been removed, some QoS examples have been condensed into a single example.

As this is not a proper issue, I am moving this ticket to the corresponding forum according to the [Fast DDS contributing guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#issues-and-support).

Edit: posting the answer even though Raul has already answered.

**Comment by paxifaer** [comment]
I get it, now I'm going to start spending some time looking at your latest example code.
